src/gui/qgslabelinggui.h

```diff
@@ -52,28 +52,12 @@ class GUI_EXPORT QgsLabelingGui : public QgsTextFormatWidget
 
     void setSettings( const QgsPalLayerSettings &settings );
 
-    /**
-     * Deactivate a field from data defined properties and update the
-     * corresponding button.
-     *
-     * \param key The property key to deactivate
-     *
-     * \since QGIS 3.0
-     */
-    void deactivateField( QgsPalLayerSettings::Property key );
```

Comment:
Is this not breaking the API?

Reply:
Ignore this, I saw it was moved to the parent class. Somehow github won't let me delete this.

Reply:
Also - that class is marked as non stable anyway